model/train.py

```diff
@@ -118,8 +118,7 @@ def compute_weighted_cross_entropy(logits,
     Tuple of scalar loss and batch normalizing factor.
   """
   if logits.ndim != targets.ndim + 1:
-    raise ValueError("Incorrect shapes. Got shape %s logits and %s targets" %
-                     (str(logits.shape), str(targets.shape)))
+    raise ValueError(f'Incorrect shapes. Got shape {str(logits.shape)} logits and {str(targets.shape)} targets')
```

Comment:
Nit: line has more than 80 characters.  You can split it up doing something like this perhaps:
```suggestion
  raise ValueError(
      f'Incorrect shapes. Got shape {str(logits.shape)} logits and '
      f'{str(targets.shape)} targets')
```